### PR TITLE
Improve wording of published version in logs

### PR DESCRIPTION
**Pull request by AliSoftware**
As per @ovitrif 's suggestion in p1643975038629449/1643970658.540429-slack-C02QANACA

### CODE REVIEW

**Comment by ovitrif** [comment]
Thanks a lot @AliSoftware for addressing this!

English is not my native language, so I'm not sure how a native english speaker would look up for the word `successfully`, but the web browser is always highlighting the word `succesfully` when I'm typing, which leads me to think the correct spelling is `successfully` ('s' is doubled in the middle of the word).

I added a comment in the internal P2s at paqN3M-mv-p2#comment-785

Sorry for suggesting additional work on this, it's not a big priority from my perspective and perhaps native english speaking folks don't use the doubled 's' in the middle of the word.

File: plugin/src/main/kotlin/com/automattic/android/ProjectExtensions.kt [hunk 1]
```diff
@@ -44,7 +44,7 @@ fun Project.printPublishedVersionNameAfterPublishTasks() {
         task.doLast {
             File(project.buildDir, "published-version.txt").writeText("${project.getExtraVersionName()}")
 
-            println("'${project.getExtraVersionName()}' is succesfully published.")
+            println("Version '${project.getExtraVersionName()}' has been published succesfully.")
```
Comment (AliSoftware) [inline plugin/src/main/kotlin/com/automattic/android/ProjectExtensions.kt]:
```suggestion
            println("Version '${project.getExtraVersionName()}' has been published successfully.")
```
thx @ovitrif for noticing the typo.

**Review by oguzkocer — APPROVED** [review]

**Comment by oguzkocer** [comment]
@AliSoftware @ovitrif Thank you for improving the wording! I am not planning to publish a new version for this change as there is likely going to be some other changes coming up for this plugin very soon. However, if you need the change as soon as possible I can do that, just let me know.

**Comment by ovitrif** [comment]
@oguzkocer It's perfectly ok for me to have these changes published whenever the next publishing is / will be scheduled.

The way I see this changeset is more like an improvement for the discoverability of the artifacts version, which most probably will only have an impact on a limited number of folks new to the project 😄

**Pull request merged**